README.md

```diff
@@ -128,8 +141,17 @@ All props accept arrays as values for mobile-first responsive styles.
 
 styled-system includes a higher order component to add style props to any component.
 
+**With `styled-components`**
+
+```jsx
+import hoc from 'styled-system/dist/styled-components/hoc'
```

Comment:
In fact, most of this lib works without React at all as well, so I could see it being useful even beyond React too.